tests/plugins/autocomplete/autocomplete.js

```diff
@@ -157,6 +162,32 @@
 			wait();
 		},
 
+		// (#3938)
+		'test navigation keybindings are registered after changing mode': function() {
```

Comment:
This test fails on IE9:

![image](https://user-images.githubusercontent.com/1061942/77760762-2f760f80-7037-11ea-8dc5-d3792076dc1a.png)

Since manual test passes it seems the fix is working correctly there. If this test is not super easy to fix there, we may just ignore it.

Reply:
It looks like IE9 has some weird problem with removing editable listeners once editable has been detached. I was able to reproduce the issue only using bender (it works fine using samples). It seems like dropping destroying autocomplete removes this issue. Destroying autocomplete is not really important in this test as it uses its own editor instance, so it's probably better than ignoring this test on IE9, WDYT?

Reply:
> Destroying autocomplete is not really important in this test as it uses its own editor instance, so it's probably better than ignoring this test on IE9, WDYT?

Makes sense for me. It's always better to have a working test :+1: